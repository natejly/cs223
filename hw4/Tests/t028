#!/bin/bash
# 2 Battlefields, 3 Entries

trap "/usr/bin/killall -q -u $USER ./Blotto 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./Blotto ]; then
  echo './Blotto is missing or not executable'
  echo './Blotto is missing or not executable' 1>&2
  exit 1
fi

$BIN/run -stderr=/dev/null ./Blotto 1 2 <$WHERE/t028.in
