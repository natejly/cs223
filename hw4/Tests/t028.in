P1,2,3
P2,3,2
P3,0,5

P1 P2
P1 P3
P2 P3
