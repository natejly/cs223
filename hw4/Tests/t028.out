P1 2.0 - P2 1.0
P3 2.0 - P1 1.0
P3 2.0 - P2 1.0
